packages/core/test/lib/commands/debug.js

```diff
@@ -26,24 +25,29 @@ describe("debug", function () {
   });
 
   describe("mergeConfigNetwork(config, options)", function () {
+    const url = "http://urlhost:1234";
+    const parsedUrl = new URL(url);
+    const expectedNetworkName = parsedUrl.host;
```

Comment:
A minor nit. The implementation and the  validation/test should not use the same function to generate expected and test values. In this case, both use `url.URL`. Consider something like this

```
    const host = 'urlhost';
    const port = 1234;
    const url = `http://${host}:${port}`;
    const expectedNetworkName = `${host}:${port}`;
```